ckan/public/base/javascript/modules/autocomplete.js

```diff
@@ -186,15 +186,21 @@ this.ckan.module('autocomplete', function (jQuery) {
      *
      * Returns a text string.
      */
-    formatResult: function (state, container, query) {
+    formatResult: function (state, container, query, escapeMarkup) {
       var term = this._lastTerm || null; // same as query.term
 
       if (container) {
         // Append the select id to the element for styling.
         container.attr('data-value', state.id);
       }
 
-      return state.text.split(term).join(term && term.bold());
+      var result = state.text.split(term).join(term && term.bold());
```

Comment:
This is awesome, thanks for submitting this fix.  I've tested and it seems to work as expected and escapes various tags, including script tags and helps prevent DOM side XSS. 

One thing I noticed. The bolding of the auto complete term is being escaped as well. A user will see `<b>search term</b>` in the autocomplete drop down. Maybe the bold styling can be removed, or if necessary it could be added after the term is escaped?

Reply:
Thank you for the review. 
In the autocomplete module of the group page, this._lastTerm is always null, so I've changed it to use query.term.  And I modified the code to call escapeMarkup first before bold markup the term.

Reply:
👍 Tested on my end, seems to work.

I've added a couple more comments just to document some of the things I noticed, not necessarily needing any action.